pkgs/development/python-modules/dnspython/default.nix

```diff
@@ -2,26 +2,32 @@
 , buildPythonPackage
 , fetchPypi
 , pythonOlder
+, setuptools-scm
 }:
 
 buildPythonPackage rec {
   pname = "dnspython";
-  version = "2.1.0";
+  version = "2.2.0rc1";
```

Comment:
I think we want to update with this update until the full release.

Reply:
I [asked](https://github.com/rthalley/dnspython/issues/742#issuecomment-1014892484) and we probably won't have to wait long.

Reply:
new version is out. I also added tests, since they do not touch the network anymore